data/json/vehicle_parts.json

```diff
@@ -3974,6 +3997,2250 @@
         "location" : "on_roof",
         "flags" : ["CIRCLE_LIGHT", "ODDTURN", "FOLDABLE"],
         "breaks_into" : [
+          {"item": "steel_lump", "min": 4, "max": 6},
```

Comment:
honestly I don't ever remember adding that. I know it says I did, bet I added the line "breaks_into" in...